rpm_spec/subpackages/manageiq-appliance

```diff
@@ -68,6 +68,15 @@ Requires: cockpit-ws
 #motd is owned by system
 %{__cp} -f %{_sysconfdir}/motd.manageiq %{_sysconfdir}/motd
 
+# run this if we are upgrading
```

Comment:
Can this comment be more descriptive on what this code is doing?

Reply:
ok, that was describing the gt 1 bit
The document for what is happening inside is described below.

I'll try and do a better job writing the documentation for the command